requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==1.8.2
+-e git+https://github.com/coolharsh55/python-opensubtitles@f37074796ee72c7efe1aa3ec2d534b4299b8fd88#egg=python_opensubtitles-master
```

Comment:
To be fair, I don't completely understand the problem but this doesn't look completely right.

Last time I checked I could install this with `pip`. But I didn't do it for a long time.